packages/ui/react-ui-theme/src/config/tailwind.ts

```diff
@@ -146,13 +146,17 @@ export const tailwindConfig = ({
               left: 0,
               width: '0%',
             },
-            '50%': {
-              left: 0,
+            '25%': {
+              left: '25%',
+              width: '50%',
+            },
+            '75%': {
+              left: '50%',
               width: '100%',
             },
             '100%': {
-              width: '0%',
               left: '100%',
+              width: '0%',
```

Comment:
I thought this PR removed the indeterminate status bar from Composer — are these changes still intentional?